### ISSUE TITLE
Release Flow 2.0.0.RC1

### ISSUE THREAD

**Issue by pleku** [vaadin/flow#5602]
Blocked by
- all components migration to Flow 2.0 #5489
- verification for all components to work in both modes #5586
- selected component beta releases:
    • App layout
    • Menu bar
    • Grid #5601 
    • Grid pro
    • Notification #5600 

Not blocked by but would be nice to have #5342 included.
